app/models/health/ed_ip_visit_file.rb

```diff
@@ -60,6 +60,8 @@ def load!
             else
               model_row[column] = value
             end
+            # Create an encounter identifier, if one does not exist, defaults to one visit per patient per day
```

Comment:
What happens if a patient ends up with two?  Say from two different ERs?

Reply:
In discussion, it was suggested that it was reasonable (in the absence of other ways to distinguish the data) to only report one admission per day. The CMT data delivers an encounter ID, so hopefully, this is just a safety net.

Reply:
So, maybe more to the point, would we ever end up with 2+ of the same identifier in a  CSV?  If so, that'd break the import.

Reply:
We need this as a way to prevent double counting visits if they appear in more than one file, so, I've added a lookup to prevent generating duplicates. Hopefully, this will never actually be exercised.